README.mediawiki

```diff
@@ -755,7 +755,7 @@ Those proposing changes should consider that ultimately consent may rest with th
 | Base32 address format for native v0-16 witness outputs
```

Comment:
Try replacing the "empty" line in 753 with `|- style="background-color: #ffffcf"`